src/legacy/core_plugins/interpreter/common/types/style.ts

```diff
@@ -17,8 +17,18 @@
  * under the License.
  */
 
-export const style = () => ({
-  name: 'style',
+import { ExpressionType } from '../../types';
+
+const name = 'style';
+
+export interface Style {
+  type: typeof name;
+  spec: any;
+  css: string;
+}
```

Comment:
I added this interface because it didn't exist yet. Is this correct?

Reply:
It exists: https://github.com/elastic/kibana/blob/master/x-pack/plugins/canvas/canvas_plugin_src/functions/types/style.ts#L72-L76

Feel free to move these up.

Reply:
looks like it's getting moved up in #37619 anyway -- that PR is likely to merge sooner, so i'll just wait for it to land and rebase this PR.